src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.cs

```diff
@@ -114,12 +131,36 @@ public IAsyncAction RenderAsync(UIElement element)
 		public IAsyncOperation<IBuffer> GetPixelsAsync()
 			=> AsyncOperation<IBuffer>.FromTask(async (op, ct) =>
 			{
-				return new Buffer(_buffer);
+				if (_buffer is null)
+				{
+					return new Buffer(Array.Empty<byte>());
+				}
+				return new Buffer(_buffer.AsMemory().Slice(0,_bufferSize));
 			});
 
 #if NOT_IMPLEMENTED
-		private static byte[] RenderAsPng(UIElement element, Size? scaledSize = null)
+		private static int RendeRenderAsBgra8_Premul(UIElement element, ref byte[]? buffer, Size? scaledSize = null)
```

Comment:
`Rende` prefix is there twice (saw this typo in multiple places, so probably a find replace issue)